assets/i18n/en.json

```diff
@@ -128,6 +128,7 @@
     "I18N_DELETE_ACCOUNT_PAGE_LIST_2_1": "Feedback threads and suggestions",
     "I18N_DELETE_ACCOUNT_PAGE_LIST_2_2": "Commits made to public explorations and collections that have other owners",
     "I18N_DELETE_ACCOUNT_PAGE_LIST_2_3": "Commits made to topics, stories, skills, and questions",
+    "IL8N_DELETE_ACCOUNT_PAGE_LABEL": "This will take you to a page where you can delete your Oppia account.",
```

Comment:
I18* (The i18n is the abbreviation for Internationalization i.e, 18 chars in between i and n [[ref](https://en.wikipedia.org/wiki/Internationalization_and_localization)])

Reply:
This has been resolved.